docs/examples/example-cascade-temperature-control.md

```diff
@@ -0,0 +1,64 @@
+# Cascade temperature control
+
+## Description
+The controller changes the setpoint of climate entities to regulate the temperature of room with increased responsiveness, accuracy, and stability.
+
+An example:
+A room has a temperature sensor and en electric heater. Given a setpoint temperature, the controller can make the sensor match the given value by adjusting the temperature settings of the electric heater. This type of control usually reacts faster and is more accurate then the alternative of simply changing the temperature settings of the heater.
+
+
+![cascade_flow](../images/node-ps-strategy-heat-capacitor-cascade-control.png)
+
+
+![cascade_example](https://deltamotion.com/support/webhelp/rmctools/Controller_Features/Control_Modes/Advanced_Control/CascadeLoop.png)
+
+The example utilizes a PID controller to provide a number between 0 and 1 to indicate a load on the heater
+
+ A cascade temperature controller is a controller which utelizes the input/setpoints of other controllers/climate entities to regulate to its own setpoint
+
+---
+## Requirements:
+> Home assistant integrated with Node-RED
+
+> The PID node installed - https://flows.nodered.org/node/node-red-contrib-pid
+
+> A climate entity
+
+## Instructions:
+> Create an "input_number" entity in Home Assistant and name it "cascade_setpoint"
```

Comment:
If you use backtick-quotes, you get a better formatting of such keywords.

Reply:
Started to use it  :-)